global/scripts/mppmodecheck.shlib

```diff
@@ -0,0 +1,52 @@
+# Functions for checking compliance with selected MPP mode for HCP Pipelines
+
+# --- Import log.shlib 
+
+if [[ "$(type -t log_Err_Abort)" != "function" ]]
+then
+    source "$HCPPIPEDIR/global/scripts/log.shlib" "$@"
+fi
+
+
+check_mpp_compliance()
+{
+
+    # --- Define variables
+
+    local MPPMode="$1"
+    local Compliance="$2"
+    local ComplianceMsg="$3"
+
+    # --- Initial information
+
+    if [ "${MPPMode}" = "LegacyStyleData" ]; then
+      log_Msg "LegacyStyleData mode requested."
+      log_Msg "NOTICE: You are using a mode that enables processing of acquisitions that do not"
+      log_Msg "        conform to the HCP specification as described in Glasser et al. (2013)!"
+      log_Msg "        Be aware that if the HCP requirements are not met, the level of data quality"
+      log_Msg "        can not be guaranteed and the Glasser et al. (2013) paper should not be used"
+      log_Msg "        in support of this workflow. A manuscript with comprehensive evaluation for"
+      log_Msg "        the Extended MPP workflow is in active preparation and should be appropriately"
+      log_Msg "        cited when published."
+      log_Msg "        "
```

Comment:
@glasserm Are you good with this "NOTICE" re the LegacyStyleData mode?  I'm worried it might "scare" off people from citing the 2013 paper at all, even if using just one "small" new feature.  (This plays into the broader issue of exactly which "mode" all the new features are going to be assigned to).

Reply:
I am okay with the wording, assuming we only put things that trigger it as those we don't want people confusing with the main HCP Pipelines.